### DISCUSSION TITLE
Template column type

### DISCUSSION THREAD

**Discussion by saenzramiro** [AntlerVC/firetable#805] [Support / Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Well, is not a problem but I have a field where I save a base64 image and current field types are not supporting the display of that image.

**Describe the solution you'd like**
A field where we can write a HTML template with the possibility to print inside that HTML the value of the field. If we can access all row values inside the template, it will be better.

Example:
`<img src="{row.image}" />`

**Describe alternatives you've considered**
Extend Image type to allow base64.

**Comment by saenzramiro** [comment]
👀

**Comment by harinij** [comment, marked as answer]
Hi @saenzramiro we didn't get back to you earlier on this, in case you are still looking for a solution for this, you can explore a combination of derivative and code field types to achieve this. You can have a derivative column type listen to any changes in the image field type, calculate the HTML code required and store it as a code column type. 

We also have some information on how you can get image field's URL and use it directly in your fields [here](https://github.com/rowyio/rowy/discussions/724)

**Reply by saenzramiro** [reply]
Hi @harinij
Thank you for your answer, I will take a look and try to solve it as you suggest.
